docs/buildah-commit.1.md

```diff
@@ -19,6 +19,12 @@ The image ID of the image that was created.  On error, 1 is returned and errno i
 
 ## OPTIONS
 
+**--add-file** *source:destination*
+
+Read the contents of the file `source` and add it to the committed image as a file at `destination`.  The new file will
+be owned by UID 0, GID 0, have 0644 permissions, and be given a current timestamp unless the **--timestamp** option is
+also specified.  This option can be specified multiple times.
+
```

Comment:
Not necessary for this PR, but did you consider making the destination optional?  I.e., if not specified, it would default to the source location?

Reply:
It hadn't occurred to me, but there's no reason it couldn't be optional.